cluster-health-check/cluster-health.sh

```diff
@@ -0,0 +1,519 @@
+#!/bin/bash
+# Minimum space needed to run the script (MB)
+SPACE=2048
+setup() {
+  TMPDIR=$(mktemp -d $MKTEMP_BASEDIR)
+  techo "Created ${TMPDIR}"
+}
+disk-space() {
+  AVAILABLE=$(df -m ${TMPDIR} | tail -n 1 | awk '{ print $4 }')
+  if [ "${AVAILABLE}" -lt "${SPACE}" ]
+    then
+      techo "${AVAILABLE} MB space free, minimum needed is ${SPACE} MB."
+      DISK_FULL=1
+  fi
+}
+verify-access() {
+  techo "Verifying cluster access"
+  if [[ ! -z $OVERRIDE_KUBECONFIG ]];
```

Comment:
Instead of using this environment variable, can you add `--kubeconfig` argument directly to the script? It would provide a better UX.